frontend/package.json

```diff
@@ -1,39 +1,28 @@
 {
-  "name": "yohoard",
+  "name": "frontend",
```

Comment:
Może "yohoard-ui"? Trochę lipa dawać po prostu "frontend". To jakby dać w gradle'u root nazwę projektu "backend" :D